source/math/IntSize2D.ooc

```diff
@@ -36,7 +36,7 @@ IntSize2D: cover {
 	minimum: func ~Int (ceiling: Int) -> This { this minimum(This new(ceiling)) }
 	maximum: func ~Int (floor: Int) -> This { this maximum(This new(floor)) }
 	clamp: func (floor, ceiling: This) -> This { This new(this width clamp(floor width, ceiling width), this height clamp(floor height, ceiling height)) }
-	fillEven: static func (other: This) -> This { This new(other width + (other width % 2 == 1 ? 1 : 0), other height + (other height % 2 == 1 ? 1 : 0)) }
+	fillEven: static func (other: This) -> This { This new(other width + (Int odd(other width) ? 1 : 0), other height + (Int odd(other height) ? 1 : 0)) }
```

Comment:
After I wrote this I've realized the `size` covers are assumed to have nonnegative values - they're not general vectors. Right? I don't know why though. In that case using `%` is just fine.